Задания с 25.02.2022/Task2/StackCalculatorTests/StackCalculatorTests.cs

```diff
@@ -0,0 +1,37 @@
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using PostfixCalculator;
+
+namespace StackCalculatorTests;
+
+public class StackCalculatorTests
+{
+    private static IEnumerable<TestCaseData> Stacks
+        => new TestCaseData[]
+        {
+        new TestCaseData(new PostfixCalculator.PostfixCalculator(new ArrayStack())),
+        new TestCaseData(new PostfixCalculator.PostfixCalculator(new ListStack()))
+        };
+
+    [TestCaseSource(nameof(Stacks))]
+    public void CalculateCorrectSequence(PostfixCalculator.PostfixCalculator calculator)
+    {
+        (double? result, bool correctWork) = calculator.CalculateExpression("225 6 7 8 + * /");
+        Assert.IsTrue(correctWork && Math.Abs((double)result - 2.5) < 0.000001);
```

Comment:
Лучше делить Assert-ы на "элементарные":
```suggestion
        Assert.IsTrue(correctWork);
        Assert.AreEqual(2.5, result.Value, 0.000001);
```